src/EventSubscriber/TideSiteRequestEventSubscriber.php

```diff
@@ -214,16 +214,16 @@ public function onRequestAddSiteFilter(GetResponseEvent $event) {
   /**
    * Add Site to cache context and tags of JSON API response.
    *
-   * @param \Symfony\Component\HttpKernel\Event\FilterResponseEvent $event
+   * @param \Symfony\Component\HttpKernel\Event\ResponseEvent $event
    *   The event object.
    */
-  public function onResponseAddSiteFilterCacheContext(FilterResponseEvent $event) {
+  public function onResponseAddSiteFilterCacheContext(ResponseEvent $event) {
     $response = $event->getResponse();
     if (!$response instanceof CacheableResponseInterface) {
       return;
     }
 
-    $site_id = $event->getRequest()->query->get('site');
+    $site_id = $event->getRequest()->query->all()['site'] ?? [];
```

Comment:
In symfony 6, the InputBag::get(), no longer accepts arrays as default values

Reply:
I used to use query->get so much in Symfony work, I have no idea why the moved it towards the fetch all as an array and get the element (probably a PHP 8 thing truth be told)